src/test/java/uk/ac/ebi/atlas/download/FileDownloadControllerWIT.java

```diff
@@ -124,6 +123,20 @@ void downloadArchiveForValidExperimentAccession() throws Exception {
                 .andExpect(content().contentType("application/zip"));
     }
 
+    @Test
+    void downloadMetadataArchiveForValidExperimentAccession() throws Exception {
+        var expectedArchiveName =
+                MessageFormat.format(
+                        ARCHIVE_NAME, EXPERIMENT_ACCESSION, ExperimentFileType.EXPERIMENT_METADATA.getId());
+
+        this.mockMvc.perform(get(ARCHIVE_DOWNLOAD_URL, EXPERIMENT_ACCESSION)
+                .param("fileType", ExperimentFileType.EXPERIMENT_METADATA.getId()))
+                .andExpect(status().isOk())
+                .andExpect(
+                        header().string(HttpHeaders.CONTENT_DISPOSITION, "attachment;filename=" + expectedArchiveName))
+                .andExpect(content().contentType("application/zip"));
+    }
+
     @Test
```

Comment:
It will be good if you can add a test case for `downloadMetadataArchiveForValidExperimentAccessions` as well.

Reply:
Thanks for the comment but the metadata download link is in single experiment page. Although it is an archive but it only for a single experiment. In the current context, we do not need a test for multiple experiment accessions.